### DISCUSSION TITLE
support Alpine.JS

### DISCUSSION THREAD

**Discussion by hai-shift** [abordage/laravel-html-min#5] [Ideas]
yout pack is grate 

plz add support to AlpineJS

many thanks

**Comment by abordage** [comment]
Hi, @hai-shift
Need more details. What exactly doesn't work or breaks?

**Comment by martinsnajdr** [comment]
I also have problem with Alpine.js. It breaks Alpine.js expression when there is no semicolon at the end of the line. BTW: I'm using Filament package. 

For example:

`Alpine Expression Error: Unexpected keyword 'if'`

`error = $el.querySelector('[data-validation-error]') if (! error) { return } isCollapsed = false if (document.body.querySelector('[data-validation-error]') !== error) { return } setTimeout(() =>$el.scrollIntoView({ behavior: 'smooth', block: 'start', inline: 'start' }), 200)`

**Comment by abordage** [comment]
@martinsnajdr, unfortunately I don't use filament and can't test. You may have the `remove_blank_lines_in_script_elements` option enabled, try turning it off.

You can also show a full example of the page that is breaking.
